scss/_buttons.scss

```diff
@@ -74,6 +74,8 @@
   }
   // End mod
 
+  // Boosted mod: no `.btn-check:checked:focus-visible + &` to handle box shadow
```

Comment:
I don't think there's anything to add on our side. Please double-check that you agree with that statement.